.gitignore

```diff
@@ -10,6 +10,9 @@
 # Compiled python code.
 *.pyc
 
+# Distribution / packaging
+*.egg-info/
```

Comment:
If you'd like to ignore files generated for distribution and packaging purposes, I'd recommend this:

    /robottelo.egg-info
    /build/
    /dist/

I've added a leading slash to make sure that only files in the root directory are ignored. If a file named e.g. `/robottelo/robottelo.egg-info` is added, then that's really unusual, and we want to know about it!